app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/OnboardingToolbarPositionPickerViewHolder.kt

```diff
@@ -45,6 +53,16 @@ class OnboardingToolbarPositionPickerViewHolder(view: View) : RecyclerView.ViewH
         }
     }
 
+    private fun setTopIllustrationSelected() {
```

Comment:
If they're in the same radio group why do we have to do this? Shouldn't they already be getting "checked" and "unchecked"? Also if we do have to do it, can we pull this into the `OnboardingRadioButton` class so we don't repeat it everywhere they are used?

Reply:
They aren't in the same radio group, it's actually a different ImageButton used for those drawables, and we want to change their borders in tandem with the radio buttons. 

But you are right, we could move the logic to the `OnboardingRadioButton` to avoid repetition!